ffsync/ordered_id_test.go

```diff
@@ -33,7 +33,7 @@ func TestUint64OrderedId(t *testing.T) {
 }
 
 func TestMemoryIdGenerator(t *testing.T) {
-	generator := NewMemoryOrderedIdGenerator()
+	generator, _ := NewMemoryOrderedIdGenerator()
```

Comment:
don't drop the error, you can't assume it'll always be this way and not drop an error

Reply:
Wasn't changed?